src/app/mork/models/users.py

```diff
@@ -0,0 +1,94 @@
+"""Mork users models."""
+
+import enum
+from typing import List
+from uuid import uuid4
+
+from sqlalchemy import (
+    Enum,
+    ForeignKey,
+    Index,
+    Integer,
+    String,
+    UniqueConstraint,
+)
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+
+from mork.models import Base
+
+
+class ServiceName(str, enum.Enum):
+    """Enum for service names."""
+
+    ASHLEY = "ashley"
+    EDX = "edx"
+    BREVO = "brevo"
+    JOANIE = "joanie"
+
+
+class DeletionReason(str, enum.Enum):
+    """Enum for the reason of the deletion request."""
+
+    USER_REQUESTED = "user_requested"
+    GDPR = "gdpr"
+
+
+class DeletionStatus(str, enum.Enum):
+    """Enum for deletion statuses."""
+
+    TO_DELETE = "to_delete"
+    DELETED = "deleted"
+
+
+class ServiceStatus(Base):
```

Comment:
I would rename the table as it concerns the user deletion status on a specific service. On first reading, when called on the `read_users` method, I thought the table concerns the table specifically.
What do you think of "UserDeletionStatus" or "UserServiceStatus"

Reply:
You're right! Choosing `UserServiceStatus`